service/telemetry.go

```diff
@@ -85,7 +81,7 @@ func init() {
 // registerInternalMetricFeatureGate registers the Internal Metric feature gate to the passed in registry
 func registerInternalMetricFeatureGate(registry *featuregate.Registry) {
 	registry.MustRegister(featuregate.Gate{
-		ID:          useOtelForInternalMetricsfeatureGateID,
+		ID:          obsreportconfig.UseOtelForInternalMetricsfeatureGateID,
```

Comment:
This registration should move to `obsreportconfig` if the ID constant is moving there as well.

Reply:
done.